src/common/models/measure/measure.ts

```diff
@@ -224,22 +232,34 @@ export class Measure implements Instance<MeasureValue, MeasureJS> {
     return this.formatFn(datum[this.name] as number);
   }
 
-  public change(propertyName: string, newValue: any): Measure {
+  public change(propertyName: string, newValue: any): this {
```

Comment:
Please see hear for real world usage: https://github.com/implydata/immutable-class/pull/1/files#diff-ae89c91267724b7637f7bb0f71f7ffdcR114

I was just trying it out here as a test, eventually this `change` method in Measure will be deleted anyways